api/src/index.js

```diff
@@ -23,9 +25,68 @@ apiRouter.get("/", async (req, res) => {
 
 // This nested router example can also be replaced with your own sub-router
 apiRouter.use("/nested", nestedRouter);
+app.use('/api/meals', mealRouter);
+app.use('/api/reservation', reservationRouter);
+
 
 app.use("/api", apiRouter);
 
 app.listen(process.env.PORT, () => {
   console.log(`API listening on port ${process.env.PORT}`);
 });
+//Future-meal
```

Comment:
I like that you are putting comments explaining what the endpoints does, but right now the comment is the same as the endpoint name:
`Future-meal`
and
`future-meal`

either remove the comments or improve them, here is an idea.
```
/* GET Future meal
Success 
(200): Returns all stored meals
Failure:
(500), unknown database error
*/
```

there are probably better naming standards than this, but the important thing is to be consistent :)